app-modernization/azuredeploy.json

```diff
@@ -0,0 +1,388 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "prefixName": {
+      "type": "string",
+      "minLength": 3,
+      "metadata": {
+        "description": "Prefix name for all components, accepts numbers and letters only."
+      }
+    },
+    "sqlServerAdminLogin": {
+      "defaultValue": "administratorLogin",
+      "minLength": 3,
+      "type": "string",
+      "metadata": {
+        "description": "Admin user name for sql server."
+      }
+    },
+    "sqlServerAdminPassword": {
+      "minLength": 6,
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password for sql server."
+      }
+    }
+  },
+  "variables": {
+    "prefixNameLower": "[toLower(parameters('prefixName'))]",
+    "webAppName": "[concat(parameters('prefixName'), 'Web')]",
```

Comment:
for resource names that are globally unique we need to generate a uniqueString for the name